tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -106,6 +107,30 @@ filings.forEach((filing: any, index: number) => {
       expect(item.correctionFilingId).toBe(filing.correctionFilingId)
     })
 
+    itIf(isIncorporationApplication(filing))('TP Hide Documents button for IAs', async () => {
```

Comment:
I think this test won't work in this test suite because this test suite looks at real filings (per the filings.json file) but you are testing a bootstrap filing.

Reply:
Agree, thank you.
Now I try to do it in FilingHistoryList1.spec.ts, with a temp case.
- The text content of the button is well tested in Unit test, it shows "Hide" and "View"
- However, it seems the "click" is not triggered in the test file, so I can't test the "View is hidden when expanded". I commented it out